src/app/routes/app-routes.tsx

```diff
@@ -153,6 +153,18 @@ function AppRoutesAfterUserHasConsented() {
           {ledgerTxSigningRoutes}
           <Route path={RouteUrls.TransactionBroadcastError} element={<BroadcastErrorDrawer />} />
         </Route>
+        <Route
+          path={RouteUrls.SendCryptoAsset}
+          element={
+            <AccountGate>
+              <Suspense fallback={<FullPageWithHeaderLoadingSpinner />}>
+                <SendCryptoAsset />
+              </Suspense>
+            </AccountGate>
+          }
+        />
+        <Route path={RouteUrls.SendCryptoAssetForm} element={<SendCryptoAssetForm />} />
+        <Route path={RouteUrls.SendCryptoAssetFormConfirmation} element={<>confirmation</>} />
```

Comment:
Moved these here @fbwoolf because of dep cruiser rules, maybe we can make a feature of it and keep the routes together instead?

Reply:
Ok, yep, ended up not being as many new routes as I had originally thought. 👍